newIDE/app/src/Export/Builds/BuildsList.js

```diff
@@ -26,6 +26,21 @@ const styles = {
   },
 };
 
+const formatBuildText = (
+  buildType: 'cordova-build' | 'electron-build' | 'web-build'
+) => {
+  switch (buildType) {
+    case 'cordova-build':
+      return <Trans>Apk Build (Cordova)</Trans>;
```

Comment:
Let's use the same terminology as in the export:

- `Android Build` 
- `Windows/macOS/Linux Build`
- `Web (upload online) Build`

(.apk is a "technical detail", might be an .aab later) (same for Cordova/Electron - it's true but less relevant for the user)